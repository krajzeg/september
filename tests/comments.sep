# This is commented out.

print("# This is not.") # But this is!
print("And semicolon insertion is not misled by comments.")
