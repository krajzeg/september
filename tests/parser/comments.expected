# This is not.
And semicolon insertion is not misled by comments.
